packages/react/src/components/SkeletonIcon/SkeletonIcon.tsx

```diff
@@ -5,32 +5,31 @@
  * LICENSE file in the root directory of this source tree.
  */
 
-import PropTypes from 'prop-types';
```

Comment:
We need to keep the `propType` definitions in so that the props are rendered properly in Storybook. They can go at the end of the file.